src/main/java/com/sda/dao/implementation/CategoryDaoImplementation.java

```diff
@@ -0,0 +1,38 @@
+package com.sda.dao.implementation;
+
+import com.sda.dao.CategoryDao;
+import com.sda.entity.Category;
+import org.hibernate.Session;
+
+import java.io.Serializable;
+
+public class CategoryDaoImplementation implements CategoryDao {
+
+    private  Session session;
+
+    @Override
+    public boolean addCategory(Category category) throws Exception {
+        Serializable save = session.save(category);
```

Comment:
Since we do not know how to use Serilizable, let's not use it in our project. In addition to this maybe we should just return void.